test/everypolitician/popolo/organization_test.rb

```diff
@@ -33,6 +49,27 @@ def test_website
     assert_equal 'http://www.irl.ee/', irl.links.first[:url]
   end
 
+  def test_image
+    assert_equal 'https://upload.wikimedia.org/wikipedia/commons/0/0c/EKRE_logo.png', ekre.image
+  end
+
+  def test_links
+    assert_equal [], eva.links
+    assert_equal 0, eva.links.count
+    assert_equal 1, ekre.links.count
```

Comment:
Per the other test cases here, I think I'd expect this to test what that link actually _is_